helper-scripts/options/balenaio.json

```diff
@@ -16,10 +18,16 @@
     "agentsMaxHoursWeek": [],
     "agentsFixHours": []
   },
-  "tracks": [
-    { "start_hour": 6, "end_hour": 27, "start_day": 0, "end_day": 4 },
-    { "start_hour": 8, "end_hour": 23, "start_day": 0, "end_day": 4 },
-    { "start_hour": 9, "end_hour": 20, "start_day": 0, "end_day": 4 },
-    { "start_hour": 12, "end_hour": 18, "start_day": 0, "end_day": 4 }
+  "hoursCoverage": [
+    {"start_day": 0, "end_day": 0, "min_hours": 53, "max_hours": 53},
+    {"start_day": 1, "end_day": 1, "min_hours": 53, "max_hours": 53},
+    {"start_day": 2, "end_day": 2, "min_hours": 53, "max_hours": 53},
+    {"start_day": 3, "end_day": 3, "min_hours": 53, "max_hours": 53},
+    {"start_day": 4, "end_day": 4, "min_hours": 53, "max_hours": 53}
```

Comment:
@AlidaOdendaal , how do we compute the values for the `min_hours` and `max_hours`? Is this per agent or is it across all of the agents scheduled for the day?

Reply:
@cmfcruz This is across all agents scheduled for the day, in this case, since `start_day` and `end_day` are equal here. More generally it is the total hours covered over the time-span indicated. So if `start_day=1` and `end_day=3`, the indicated number of hours would be a total for Tuesday through Thursday, without the need for equal spreading between these 3 days.